ci/cloudbuild/builds/lib/cmake.sh

```diff
@@ -36,13 +36,13 @@ export NINJA_STATUS="T+%es [%f/%t] "
 # This block is run the first (and only) time this script is sourced. It first
 # clears the ccache stats. Then it registers an exit handler that will display
```

Comment:
Seeing as you changed "ccache" to "sccache" in one log message below, I imagine you also want to change it in other log messages and comments too?

Reply:
Fixed